test/cypress/integration/test.acceptance.js

```diff
@@ -140,11 +141,6 @@ describe('Shepherd Acceptance Tests', () => {
           .contains('Shepherd is a JavaScript library')
           .should('exist')
           .and('be.visible');
-
-        // Step twp text should be hidden
-        cy.contains('Including Shepherd is easy!')
```

Comment:
I think it was wrong when I [added it in the first place](https://github.com/shipshapecode/shepherd/commit/a473c24827cf9834094a428e3321205fbf7abc95).

The hidden steps souldn't exist in the DOM at all. And also, [Cy works in a way](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html) that it will try to wait until it finds something -- which causes it to hang here.

Reply:
Unless we changed something, hidden steps definitely should exist in the DOM. Once a step is created and inserted into the DOM, it remains there, but is just set to `hidden=true`. In this case, it may not have been inserted into the DOM yet, but if that is the case, we should do a couple next/back clicks to make sure it is inserted into the DOM, and hidden when on another step. Not sure if we already have tests covering that or not.

Reply:
We set `this.el` to `hidden`, but then we also destroy the tippy instance. Since the former is contained in the later, it gets removed from the DOM:

```js
  hide() {
    this.trigger('before-hide');

    if (this.el) {
      this.el.hidden = true;
      // We need to manually set styles for < IE11 support
      this.el.style.display = 'none';
    }

    document.body.('data-shepherd-step');

    if (this.target) {
      this.target.classList.remove('shepherd-enabled', 'shepherd-target');
    }

    if (this.tooltip) {
      this.tooltip.hide();
    }

    this.trigger('hide');
  }
```